gatsby/src/components/BackToTop/BackToTop.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from "react";
+
+import styles from "./BackToTop.module.scss";
+
+interface BackToTopProps {
+	scrollTo?: string;
```

Comment:
This is prsumably an id not a y pos or anything? Maybe call it scrollTargetId or something?